docs-v2/lib/dirToTree.js

```diff
@@ -35,30 +38,33 @@ const createFile = (stat, fileName, workingDir, baseDir) => {
       name,
       title,
       isFile: true,
-      route: path.join(routeBase, routeFile),
+      route: path.posix.join(routeBase, routeFile),
```

Comment:
Can you give some context on this?

Reply:
in windows all the routes were generated with the "\" instead of "/" and it breaks routing completely.

Reply:
This works on windows, though? Interesting.

Reply:
Yeah, it just enforce the `path` to use posix style thingies..